docs/modules/components/pages/outputs/cassandra.adoc

```diff
@@ -79,6 +80,11 @@ output:
     host_selection_policy:
       local_dc: "" # No default (optional)
       local_rack: "" # No default (optional)
+    reconnect_interval: 60s
+    exponential_reconnection:
+      reconnection_max_retries: 0 # No default (required)
+      reconnection_initial_interval: "" # No default (required)
+      reconnection_max_interval: "" # No default (required)
```

Comment:
I wonder if we have `exponential_reconnection` perhaps the `reconnection` prefix in the fields could be skipped.